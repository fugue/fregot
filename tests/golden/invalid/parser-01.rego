package fregot.tests.invalid.parser_01

# This is a common mistake and the error is bad since `function` is seen as
# a boolean rule with no body.
function double(x) = y {
  y = x * 2
}
