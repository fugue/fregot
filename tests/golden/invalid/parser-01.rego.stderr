fregot (compile error):
  "parser-01.rego" (line 5, column 1):
  incomplete rule definition:

    5| function double(x) = y {
       ^^^^^^^^

  Rules cannot be declared using just their name.
  The rule function should have an index, arguments or a value.
